File: app/src/main/java/com/github/ovictorpinto/verdinho/ui/main/SobreFragment.java
package com.github.ovictorpinto.verdinho.ui.main;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.github.ovictorpinto.verdinho.R;

public class SobreFragment extends Fragment {

    public SobreFragment() {
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        getActivity().setTitle(R.string.informacoes);
        return inflater.inflate(R.layout.ly_sobre, null);
    }
}

File: app/src/main/java/com/github/ovictorpinto/verdinho/ui/main/SobreFragment.kt
package com.github.ovictorpinto.verdinho.ui.main

import android.app.Dialog
import android.app.DialogFragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.view.Window

import com.github.ovictorpinto.verdinho.R
import kotlinx.android.synthetic.main.ly_sobre.view.*

class SobreFragment : DialogFragment() {

    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        val dialog = super.onCreateDialog(savedInstanceState)
        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE)
        return dialog
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        var view = inflater.inflate(R.layout.ly_sobre, null)
        view.bt_close.setOnClickListener({ dismiss() })
        return view
    }
}
